### PR TITLE
bugfix alpine glacier

### PR DESCRIPTION
**Pull request by luraess**
The `fetch_glacier` function was broken wrt alpine glacier selection. This PR fixes following:
- capitalise `OST` and `WEST` to match downloaded swissTLM3D data file names
- use `joinpath` for more robust path handling
- use `DimPoints` to retrieve DEM coordinates
- use updated Rasters.jl to fix `mosaic` function 
- HDF5 I/O

### CODE REVIEW

File: src/Alpine_glaciers.jl [hunk 1]
```diff
@@ -17,27 +17,28 @@ function fetch_glacier(name::String, SGI_ID::String; datadir::String)
...
 function organise_folder(dir)
-    files_to_move = [dir * "04_IceThickness_SwissAlps/IceThickness.tif"
-                     dir * readdir(dir * "08_SurfaceElevation_SwissAlps/")
-                     dir * "TLM_BB/" .* filter(x -> startswith(x,"swissTLM3D_TLM_BODEN") && (endswith(x,".shp") || endswith(x,".dbf")), readdir(dir*"TLM_BB/"))]
-    dirs_to_remove = dir .* filter(x->isdir(dir*x),readdir(dir))
+    files_to_move = [joinpath.(dir,"04_IceThickness_SwissAlps/IceThickness.tif")
+                     joinpath.(dir,"08_SurfaceElevation_SwissAlps/",readdir(joinpath(dir,"08_SurfaceElevation_SwissAlps/")))
+                     joinpath.(dir,"TLM_BB", filter(x->startswith(x,"swissTLM3D_TLM_BODEN") && (endswith(x,".shp") || endswith(x,".dbf")),readdir(joinpath(dir,"TLM_BB/"))))
+                     joinpath.(dir,"TLM_BB", filter(x->startswith(x,"swissTLM3D_TLM_GLAMOS") && endswith(x,".dbf"),readdir(joinpath(dir,"TLM_BB/"))))]
+    dirs_to_remove = joinpath.(dir,filter(x->isdir(dir*x),readdir(dir)))
```
Comment (mauro3) [inline src/Alpine_glaciers.jl]: use `joinpath` here as well

**Comment by luraess** [comment]
Actually still not OK as it now hangs on `mask(rasterDat; with=poly)` ...

**Review by mauro3 — COMMENTED** [review]
Some tests would be good too.  Especially as Rasters is a moving target.

**Comment by mauro3** [comment]
Have a look at the code.  I think that does what you want it to do.

**Comment by luraess** [comment]
> Have a look at the code. I think that does what you want it to do.

Indeed - that does what I want. Thanks

**Comment by mauro3** [comment]
Note that the padding is now in meters and not gridpoints. (oops: edit "not" -> "now")

**Comment by luraess** [comment]
Will try to add some tests.

**Comment by mauro3** [comment]
Merge this now?

**Comment by luraess** [comment]
I'd say so, and I'll open a new issue to add tests.

**Pull request merged**